### DISCUSSION TITLE
Can't seem to install WindowsInk

### DISCUSSION THREAD

**Discussion by saviopham-03** [OpenTabletDriver/OpenTabletDriver#1964] [Q&A] [answered]
I just opened up the program using the specified instructions, but I've been having trouble installing WindowsInk.

According to the WindowsInk wiki, installing should come with OpenTabletDriver in the Plugins section, however I can't seem to find it there.
<img width="722" alt="no windows ink" src="https://user-images.githubusercontent.com/41348578/149655500-e40b30fa-d65f-4847-9b8c-d98718904301.png">

So, I tried downloading WindowsInk from here https://opentabletdriver.net/Plugins and dragging the .dll files in, but this happens
<img width="704" alt="Screenshot_8" src="https://user-images.githubusercontent.com/41348578/149655729-10a0e0e0-97fa-4895-89c1-16fa7b7111fd.png"> (I've also tried dragging the whole .zip file in with the same outcome)

I'm not sure what to do, and if its worth mentioning, everytime I open it, I get a notification that theres an update, but when I try to update, it remains as version 6.0.0, and I get the notification that there's an update again. Also, every time I open it, I get the OpenTabletDriver Guide startup explanation popup, regardless of whether or not its my first time opening it or my 30th. I have also already installed the VMulti Driver but this still happens.

Thanks.

**Comment by jamesbt365** [comment, marked as answer]
https://github.com/X9VoiD/VoiDPlugins/actions/runs/1688996928
